File: src/Services/WareHouse/WareHouse.API/Application/Model/SelectTopDashBoardDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WareHouse.API.Application.Model
{
    public class SelectTopDashBoardDTO
    {
        public DashBoardSelectTopInAndOut ItemCountMax { get; set; }
        public DashBoardSelectTopInAndOut ItemCountMin { get; set; }
        public SelectTopWareHouseDTO WareHouseBeginningCountMax { get; set; }
        public SelectTopWareHouseDTO WareHouseBeginningCountMin { get; set; }
    }
}

File: src/WebClient/WareHouse/src/app/model/SelectTopDashBoardDTO.ts
import { DashBoardSelectTopInAndOutDTO } from "./DashBoardSelectTopInAndOutDTO";
import { SelectTopWareHouseDTO } from "./SelectTopWareHouseDTO";

export interface SelectTopDashBoardDTO {
    itemCountMax: DashBoardSelectTopInAndOutDTO|undefined;
    itemCountMin: DashBoardSelectTopInAndOutDTO|undefined;
    wareHouseBeginningCountMax: SelectTopWareHouseDTO|undefined;
    wareHouseBeginningCountMin: SelectTopWareHouseDTO|undefined;
}